plugins/reanimated.js

```diff
@@ -0,0 +1,73 @@
+const { createRunOncePlugin } = require("@expo/config-plugins");
+const {
+  mergeContents,
+} = require("@expo/config-plugins/build/utils/generateCode");
+const { withBridgeDelegate } = require("./index");
+
+const NAME = "react-native-reanimated";
+
+function addContents(tag, src, newSrc, anchor) {
```

Comment:
The plugin isn't written in TS due to the overhead of dealing with the transpiling step here?

Reply:
In this repo, we use [TSDoc](https://tsdoc.org/) + [`@ts-check`](https://www.typescriptlang.org/docs/handbook/intro-to-js-ts.html), and run `tsc` on CI. So we do type checking, but not transpilation. Mostly to make it easier to debug and hack on the consumer side. I will add types and do some more testing before making this ready for review.